### ISSUE TITLE
Continuity from main region and start of the real content is broken 

### ISSUE THREAD

**Issue by wilsmic2** [ManifoldScholar/manifold#2866]
For many pages in Manifold, specifically ones that have the secondary navigation region in them, there is an incorrect formatting of the page using landmarks that can be seen. The main region of the page should start with a heading 1 and be followed by relevant content, but at this time there are many objects between the start of the region and the main content as well as object between the heading 1 and the main content. Screen reader users have to travel through multiple object including secondary navigation bar, the preview, delete, and thumbnail objects, and the project setting navigation to reach the real main content of the page. I will describe this in more detail below with visual examples as well as what screen reader users see when they interact with this webpage. To be clear, this issue can occur on any page that has the secondary navigation bar, but we will be looking specifically at the project edit pages that are found in admin mode.

## Current Behavior
![Manifold Project Edit Page](https://user-images.githubusercontent.com/50855208/95513376-1b5c8480-096f-11eb-8056-56d58c8eae24.png)

With this image we can see how the page is currently laid out in terms of landmarks and headings, where it is important to note that the "Secondary Navigation Bar" that I will be referrering to is the navigation bar at the top of the main content that has "All Projects" and "Project Collections". Even when looking at this page visually, we can see that the main content area does not start with a Heading 1, but instead starts with the secondary navigation bar, which can be very confusing to screen reader. In addition to this, considering the information is presented from top to bottom, left to right, we can see that after the Header 1 is the "preview" and "delete" buttons, as well as the project settings navigation bar, which is the one on the left side of the screen with General, Layout, Access, etc. before the actual main content of the page, which starts with the General Settings. This layout of not having the main content area begin with an H1 header as well as there being content between the H1 header and the main content can become quite difficult for screen reader user to navigate since when they go to the main content of the page, they expect a header 1 followed by the main content. I hope that this output from the screen reader that I am showing below will help highlight the issue. Please let me know if you have any questions.

```
[1]  University of Washington
[2]  SKIP TO MAIN CONTENT
[3]  Skipped to Main Content
[4]  Return to home
[5]  figure
[6]  Manifold Logo
[7]  figure end
[8]  Primary Navigation navigation region
[9]  Page Links list of 3 items
[10] Dashboard
[11] Projects
[12] Records
[13] list end
[14] Primary Navigation navigation region end
[15] navigation region
[16] User Links list of 2 items
[17] EXIT ADMIN MODE
[18] User settings
[19] list end
[20] navigation region end

[21] main region             # Start of Main Region

[22] Secondary Navigation navigation region
[23] list of 2 items
[24] All Projects
[25] Project Collections
[26] list end
[27] Secondary Navigation navigation region end

[28] UW Test Project          # Header 1

[29] PREVIEW
[30] DELETE
[31] figure
[32] Default Project Thumbnail
[33] figure end

[34] Project Settings navigation region
[35] list of 12 items
[36] General
[37] Layout
[38] Access
[39] People
[40] Texts
[41] Resources
[42] Resource Collections
[43] Activity
[44] Metadata
[45] Social Integrations
[46] Exports
[47] Log
[48] list end
[49] Project Settings navigation region end

[50] group start GENERAL       # Beginning of actual main content
[51] GENERAL
[52] TITLE
[53] UW Test Project
[54] SUBTITLE
[55] Enter Project Subtitle
[56] PUBLICATION DATE
...
```

With this screen reader view, we can see a more detailed look at the issue for screen reader users. We can see that the main region begins on line 21, where lines 22-27 consist of the secondary navigation, and the Header 1 appears on line 28. we then see that lines 29-49 consist of the preview, delete, project thumbnail, and project settings navigation region, where the main content actually begins on line 50. For screen reader users, this is a lot of information to tab through, especially if they are trying to discover the main layout and content of the webpage.

## Expected Behavior/Possible Solutions
Ideally, through even possible reconstruction of the page, both navigation regions would be removed from the main content area, and the main content area would start with the header 1 followed by the main content. We realize that this becomes a much larger task if restructuring like this were to occur, so we have brainstormed some solutions that will help to alleviate some of the difficulties that are encountered by screen reader users.
 <ol>
<li> Preferably eliminate the secondary navigation region (especially since the only functionality that it is has is allowing navigation to project collections, as the "All Projects" link takes you to the same page as the "Projects" link in the main navigation region) or at least move it to outside the main region.</li>
<li> Move the Preview and Delete buttons to the end of the form after the Save button. This is mainly because you only really need to preview/delete the project after it has been created/edited, meaning that it would make more logical sense to have these buttons near the save button located at the bottom of the page.</li>
<li>We are not sure if the thumbnail is a very useful thing to have as users have to preview the project to check its content or layout so you might remove it altogether. If you think it is an important feature that should remain, please set the alt text to null so that it is not rendered at all for the screen reader.</li>
</ol>

Here is an example of what the tab ordering could look like if these changes were applied:
```
[1]  University of Washington
[2]  SKIP TO MAIN CONTENT
[3]  Skipped to Main Content
[4]  Return to home
[5]  figure
[6]  Manifold Logo
[7]  figure end
[8]  Primary Navigation navigation region
[9]  Page Links list of 3 items
[10] Dashboard
[11] Projects
[12] Records
[13] list end
[14] Primary Navigation navigation region end
[15] navigation region
[16] User Links list of 2 items
[17] EXIT ADMIN MODE
[18] User settings
[19] list end
[20] navigation region end

[21] Secondary Navigation navigation region
[22] list of 2 items
[23] All Projects
[24] Project Collections
[25] list end
[26] Secondary Navigation navigation region end

[27] main region             # Start of Main Region
[28] UW Test Project          # Header 1

[29] Project Settings navigation region
[30] list of 12 items
[31] General
[32] Layout
[33] Access
[34] People
[35] Texts
[36] Resources
[37] Resource Collections
[38] Activity
[39] Metadata
[40] Social Integrations
[41] Exports
[42] Log
[43] list end
[44] Project Settings navigation region end

[45] group start GENERAL       # Beginning of actual main content
[46] GENERAL
[47] TITLE
[48] UW Test Project
[49] SUBTITLE
[50] Enter Project Subtitle
[51] PUBLICATION DATE
...
[...] Save Project
[...] PREVIEW
[...] DELETE
```

If you have any questions please do not hesitate to ask. The solution above is a tentative one that could be improved even further depending on various development limitations.

## Steps to Reproduce
1. Enter the Admin Mode of the application
2. Navigate to the "Projects" tab from the main navigation area
3. Search "UW test Project" (or go any of the projects there)
4. Enter a project page and view the landmark/heading structure

**Comment by zdavis** [comment]
@bot bug

**Comment by zdavis** [comment]
Thank you for taking the time to report this bug. We've reviewed the issue and agree that this should be fixed. We’re adding a “planned” label to indicate that we consider this bug as part of our current work plan.

_This was an automated message, but please don't hesitate to reply. Our team watches these issues closely and will respond as soon as we're able to!_

**Comment by dananjohnson** [comment]
Hi @wilsmic2 👋 ! Thanks for your thorough bug report. Per your recommendation, I've implemented a fix that removes the Secondary Navigation component from the Main region. I've also removed the alt-text from the default project thumbnail.

At this time, I don't think it makes sense to pursue suggestions 1 and 2. The Secondary Navigation component is an important part of the backend UI that's used to provide contextual links to users. Hopefully removing it from the Main region is sufficient to help limit clutter for screen reader users. Suggestion 2 would require some design work and doesn't seem immediately relevant to the issue of page landmarks.
